### ISSUE TITLE
Possibe to crawl and scrape text files?

### ISSUE THREAD

**Issue by prologic** [gocolly/colly#572]
I don't see an API for this besides `Collector.OnResponse`. Curious if its possible to scrap `plain/text` files? My use-case is scraping a bunch of hosted text files that _may_ have linked to other text files.

**Comment by prologic** [comment]
Ping?

**Comment by WGH-** [comment]
What's your expectations? You want colly to find URLs in plain text using some heuristics?

**Comment by prologic** [comment]
> What's your expectations? You want colly to find URLs in plain text using some heuristics?

Yes. I want the same semantics that the colly API provides for HTML, but for plain text. Look for (_perhaps using regexes?_) things that might look like links to other documents.

In my case I'm wanting to scrape [twtxt](https://twtxt.readthedocs.org) feeds looking for other links of the form `<nick url>`. I _realize_ this isn't exactly a HTML ink pe rse, but if the API could provide a custom search so I can define what to look for as "links" in documents?

**Comment by WGH-** [comment]
What's exactly wrong with doing such regex heuristics in `OnResponse` yourself?

**Comment by prologic** [comment]
This is what i did

**Comment by WGH-** [comment]
Strictly speaking, Colly doesn't really define what's an "HTML link" either: it's up to user to enqueue more URLs in his own custom `c.OnHTML("a[href]", ...)` handler.

I personally don't think there's a merit of incorporating such site-specific heuristics into Colly codebase, but let's see if there're other opinions about this.
